src/program/command_mode.rs

```diff
@@ -67,27 +68,37 @@ impl FromStr for Commands {
     }
 }
 
-/// Converts the provided path by user to a path that can be glob'd
-/// Directories are changed from /home/etc to /home/etc/*
-fn convert_path_to_globable(path: &str) -> Result<String, String> {
+// Sanitize path by substituting env vars and removing escaped spaces
+fn sanitize_path(path: &str, base_dir: &PathBuf) -> Result<PathBuf, String> {
     let expanded_path = full(path).map_err(|e| format!("\"{}\": {}", e.var_name, e.cause))?;
     // remove escaped spaces
-    let absolute_path = String::from(expanded_path).replace(r"\ ", " ");
-    // If path is a dir, add /* to glob
-    let mut pathbuf = PathBuf::from(&absolute_path);
-    if pathbuf.is_dir() {
-        pathbuf = pathbuf.join("*");
+    let non_escaped_path = String::from(expanded_path).replace(r"\ ", " ");
```

Comment:
What about windows? How does it represent spaces in filenames?

Reply:
@Davejkane 
A horrendous mess. Simple for spaces on windows is quoting the whole path.

But others like `$` or `%` see the accepted answer.
https://stackoverflow.com/questions/30620876/how-to-properly-escape-filenames-in-windows-cmd-exe

Reply:
Though, `glob` and the stdlib seem to paper over the platform details.

Ex. "./Down loads/image.png" 
It does not require escaping.

Reply:
The command mode only escapes spaces `\ ` because we use spaces as the end of an argument.

Reply:
@gurgalex, ok, so this should be fine for windows then. Did you get a chance to try this out on windows yet?

Reply:
Didn't try yet. I'll report back after a reboot (30min to 1 hr)

Reply:
5GB of space for C++ build tools for rust installation.

We need to find a way to build standalone Windows executable so users don't need this.

Reply:
@NickHackman @Davejkane 

It compiles and finds images in the current directory of the launched program. 
I think globs are broken on linux as well though. The commit with custom normalize worked fine though.

newglobs don't seem to work on Windows.
`ng *.png` 

```
[src\program\command_mode.rs:91] &sanitized_dir = "C:\\Users\\alex\\github\\riv\\*.png"
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: "The filename, directory name, or volume label syntax is incorrect. (os error 123)"'
```

A cli glob does not work because Windows NTFS does not support ` / \ : * ? " < > | ` in filenames or directories.

So we'll need support for a quoted glob anyway it seems for a cli argument.

Reply:
@gurgalex, eventually we will look at build scripts for each platform and package manager. Homebrew, dmg, pacman, apt-get, yum, rpm, whatever windows uses. But that's still some way away as far as I see it. Maybe a `1.0` goal.

This globbing not working on linux or windows is a bit of a headache. Is that just for this branch, yeah?

Reply:
Correct, globbing is not working for just this branch.

Strangely, Escaping spaces `\ ` works on Linux, but not Windows in the `ng` command.

Reply:
@gurgalex what time zone are you in? Do you not sleep?

Reply:
Summer break, so it's easy to work into the night.

US Central (CST/CDT)